src/Simplex/Chat/Messages.hs

```diff
@@ -809,6 +813,8 @@ ciContentToText = \case
   CISndConnEvent event -> sndConnEventToText event
   CIRcvChatFeature feature enabled param -> chatFeatureToText feature <> ": " <> prefToText enabled param
   CISndChatFeature feature enabled param -> chatFeatureToText feature <> ": " <> prefToText enabled param
+  CIRcvFeatureOffer feature param -> "allow " <> featureParamText feature param <> "?"
+  CISndFeatureOffer feature param -> "offered " <> featureParamText feature param
```

Comment:
```suggestion
  CIRcvFeatureOffer feature param -> "offered " <> featureParamText feature param
  CISndFeatureOffer feature param -> "you offered " <> featureParamText feature param
```

Reply:
why not use prefParamText instead of featureParamText? "off" on Nothing param is unclear, it will be something like this:
"alice> offered disappearing messages (off)" - what off means here?
or w/t change from "allow?" - even more strange:
"alice> allow disappearing messages (off)?"